packages/dendron-plugin-views/src/components/DendronApp.tsx

```diff
@@ -101,11 +99,6 @@ function DendronVSCodeApp({ Component }: { Component: DendronComponent }) {
     }
   });
 
-  // don't load until active
-  if (_.isEmpty(engine.notes)) {
```

Comment:
I'll probably amend the logic around here - it still takes some time to render large notes, and the "Loading..." placeholder is currently gone.